O365/message.py

```diff
@@ -756,6 +762,24 @@ def reply(self, to_all=True):
 
         message = response.json()
 
+        # adding new feacture to the timezone 
+        html_content = message['body']['content']
+
+        pattern = re.compile(r'<b>Sent:</b> (.*?)<br>')
+
+        def change_sent_time(match):
+            sent_time_str = match.group(1)
+            sent_time = datetime.strptime(sent_time_str, '%A, %B %d, %Y %I:%M:%S %p')
+            # asign the time zone for example bogota
+            timezone_tz = pytz.timezone('America/Bogota')
+            sent_time_new = sent_time.replace(tzinfo=pytz.utc).astimezone(timezone_tz)
+            # return the new value
+            return f'<b>Sent:</b> {sent_time_new.strftime("%A, %B %d, %Y %I:%M:%S %p")}<br>'
```

Comment:
To maintain support of older Python versions this library doesn't use `f-strings`, rewrite this using `.format()` instead

Reply:
That's no longer True. Python 3.9 it's the oldest version possible